### DISCUSSION TITLE
Is there a way to display a structured list of items plus details for each item?

### DISCUSSION THREAD

**Discussion by Tony20221** [oqtane/oqtane.framework#4475] [Q&A]
I want to display a list of cars for sale. One page displays a list of them. Each item (car entity) will display a photo of the car and some properties (same for each car) of the car. Clicking on a car gets more details about the car including a carousel of images. All data resides in the database.
Can Oqtane with an existing third party module do this? Or do I have to create my own module?

**Comment by leigh-pointer** [comment]
@Tony20221,
There is a module that can do this for you.  Check out the DynamicForm module in the Oqtane Marketplace.  Check out https://www.studio-elf.net/dynamicform and https://www.studio-elf.net/dynamicform/only-dynamicform.  The module has two variants, a light version and a paid version.  For more information email me from here https://www.studio-elf.net/contact <- also a DynamicForm by the way!
